components/raftstore/src/store/snap.rs

```diff
@@ -1482,6 +1501,44 @@ impl SnapManager {
         Ok(v)
     }
 
+    pub fn get_temp_path_for_build(&self, region_id: u64) -> PathBuf {
+        let sst_id = self.core.temp_sst_id.fetch_add(1, Ordering::SeqCst);
+        let filename = format!(
+            "{}_{}_{}{}",
+            SNAP_GEN_PREFIX, region_id, sst_id, TMP_FILE_SUFFIX
+        );
+        PathBuf::from(&self.core.base).join(&filename)
+    }
+
+    pub fn get_final_name_for_recv(&self, key: &SnapKey) -> PathBuf {
```

Comment:
use tablet snap mgr.

Reply:
yes, I will create new runner to response for the tablet snapshot send and recv. The mixed will drop.